### ISSUE TITLE
Support for_each on resources

### ISSUE THREAD

**Issue by VladRassokhin** [VladRassokhin/intellij-hcl#235]
Will be released in Terraform 0.12.6
[Related PR in Terraform repo](https://github.com/hashicorp/terraform/pull/21922)

**Comment by VladRassokhin** [comment]
Fixed, will be supported in 0.7.6

**Comment by bkmeneguello** [comment]
will this support `each.key` and `each.value` keywords? they aren't recognized.
suggestion: your issues could be linked to commits or PRs, this could make hard to follow the changes in source code.

**Comment by VladRassokhin** [comment]
@bkmeneguello should be fixed in 0.7.6 (not released yet)
>suggestion: your issues could be linked to commits or PRs, this could make hard to follow the changes in source code.

It's impossible to link something from non-public code ;)
